src/test/integration/IntegrationChecks.t.sol

```diff
@@ -189,8 +189,9 @@ contract IntegrationCheckUtils is IntegrationBase {
         assertEq(address(operator), delegationManager.delegatedTo(address(staker)), "staker should be delegated to operator");
         assert_HasExpectedShares(staker, strategies, shares, "staker should still have expected shares after delegating");
         assert_Snap_Unchanged_Staker_DepositShares(staker, "staker shares should be unchanged after delegating");
-        assert_Snap_Unchanged_Staker_WithdrawableShares(staker, "withdrawable shares should be unchanged after delegating");
-        assert_Snap_Added_OperatorShares(operator, strategies, shares, "operator should have received shares");
+        //assert_Snap_Unchanged_Staker_WithdrawableShares_Delegation(staker, "withdrawable shares should be unchanged after delegating");
```

Comment:
Uncommenting this doesn't cause any tests to fail. Is there a reason it's commented out?

Reply:
Re-added here: [86c255d](https://github.com/Layr-Labs/eigenlayer-contracts/pull/1045/commits/86c255d601f1d7c96dc7428c8f2a3aee47ba5f0b)

I know this can be problematic because we dont have good handling for rounding issues -- i have had to tweak this in a branch i'm working on. But I'm gonna leave it uncommented unless it's actively breaking. That way when it does break, we can fix it :)